tests/_utils/view.js

```diff
@@ -0,0 +1,836 @@
+ *		getData( container, null, { showType: true } ); // '<container:p></container:p>'
+ *
+ * If `options.showPriority` is set to `true`, priority will be displayed for all
+ * {@link engine.treeView.AttributeElement AttributeElements}.
+ *
+ *		const attribute = new AttributeElement( 'b' );
+ *		attribute.priority = 20;
+ *		getData( attribute, null, { showPriority: true } ); // <b:20></b:20>
+ *
+ * @param {engine.treeView.Text|engine.treeView.Element|engine.treeView.DocumentFragment} node Node to stringify.
+ * @param {engine.treeView.Selection|engine.treeView.Position|engine.treeView.Range} [selectionOrPositionOrRange = null ]
+ * Selection instance which ranges will be included in returned string data. If Range instance is provided - it will be
+ * converted to selection containing this range. If Position instance is provided - it will be converted to selection
+ * containing one range collapsed at this position.
+ * @param {Object} [options] Object with additional options.
+ * @param {Boolean} [options.showType=false] When set to `true` type of elements will be printed (`<container:p>`
+ * instead of `<p>` and `<attribute:b>` instead of `<b>`).
+ * @param {Boolean} [options.showPriority=false] When set to `true` AttributeElement's priority will be printed
+ * (`<span:12>`, `<b:10>`).
+ * @returns {String} HTML-like string representing the view.
+ */
+export function stringify( node, selectionOrPositionOrRange = null, options = {} ) {
+	let selection;
+
+	if ( selectionOrPositionOrRange instanceof Position ) {
+		selection = new Selection();
+		selection.addRange( new Range( selectionOrPositionOrRange, selectionOrPositionOrRange ) );
+	} else if ( selectionOrPositionOrRange instanceof Range ) {
+		selection = new Selection( );
+		selection.addRange( selectionOrPositionOrRange );
+	} else {
+		selection = selectionOrPositionOrRange;
+	}
+
+	const viewStringify = new ViewStringify( node, selection, options );
+
+	return viewStringify.stringify();
+}
+
+/**
+ * Parses HTML-like string and returns view tree nodes.
+ * Simple string will be converted to {@link engine.treeView.Text Text} node:
+ *
+ *		parse( 'foobar' ); // Returns instance of Text.
+ *
+ * {@link engine.treeView.Element Elements} will be parsed with attributes an children:
+ *
+ *		parse( '<b name="baz">foobar</b>' ); // Returns instance of Element with `baz` attribute and text child node.
+ *
+ * Multiple nodes provided on root level will be converted to {@link engine.treeView.DocumentFragment DocumentFragment}:
+ *
+ *		parse( '<b>foo</b><i>bar</i>' ); // Returns DocumentFragment with two child elements.
+ *
+ * Method can parse multiple {@link engine.treeView.Range ranges} provided in string data and return
+ * {@link engine.treeView.Selection Selection} instance containing these ranges. Ranges placed inside
+ * {@link engine.treeView.Text Text} nodes should be marked using `{` and `}` brackets:
+ *
+ * 		const { text, selection } = parse( 'f{ooba}r' );
```

Comment:
I see here a different indentation. There should be two tabs (no spaces) in all of those code snippets.